docs/HyBIG-Example-Usage.ipynb

```diff
@@ -14,14 +14,17 @@
     "* **JIRA:**  [SDPS Data Services](https://bugs.earthdata.nasa.gov/secure/RapidBoard.jspa?rapidView=757&view=planning.nodetail)\n",
     "\n",
     "\n",
-    "HyBIG converts GeoTIFF inputs to Global Imagery Browse Services ([GIBS](https://nasa-gibs.github.io/gibs-api-docs/)) compatible PNG or JPEG outputs with metadata. This includes, where necessary, conversion to a GIBS supported Coordinate Reference System (CRS) and preferred resolutions. User customizations, including `crs`, `scale_extents`, and dimensions (`height` & `width`), can be requested, but the generated outputs may not be compatible with GIBS. \n",
+    "HyBIG converts GeoTIFF inputs to Global Imagery Browse Services ([GIBS](https://nasa-gibs.github.io/gibs-api-docs/)) compatible PNG or JPEG outputs with metadata. This includes, where necessary, conversion to a GIBS supported Coordinate Reference System (CRS) and preferred resolutions. User customizations, including `crs`, `scale_extents`, and dimensions (via `height` & `width`), can be requested, but note the generated outputs may not be compatible with GIBS. \n",
+    "\n",
+    "Information on how HyBIG computes default values can be found in the [README.md](https://github.com/nasa/harmony-browse-image-generator/blob/main/README.md) file of this repository.\n",
```

Comment:
Yeah - I think we needed something for someone running the notebook without looking at the README. This probably covers it, thanks!